pkg/raw/raw_exec.go

```diff
@@ -5,44 +5,101 @@ package raw
 
 import (
 	"fmt"
-	"os/exec"
-	"strconv"
+	"io"
+	"os"
 
+	"github.com/go-logr/logr"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-type Exec struct {
-}
+type Exec struct{}
+
+const filePerm = 0644
 
 func (Exec) Create(filename string, opts ...CreateOption) error {
 	o := &CreateOptions{}
 	o.ApplyOptions(opts)
-	var seek string
-
-	if o.SourceFile == "" && o.Size == nil {
-		return fmt.Errorf("must specify Size when creating without source file")
-	}
+	log := ctrl.Log
```

Comment:
optional: maybe we can create here logger with name raw-disk or something. It can  be parent logger for both createEmpty and copyFile loggers after. So we will see these log records have some bond.

Reply:
added WithName raw-disk